static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -120,7 +120,7 @@ export const init = (): Promise<void> => {
                 let npaFlag: boolean;
                 if (typeof state.tcfv2 !== 'undefined') {
                     // TCFv2 mode,
-                    npaFlag = Object.values(state.tcfv2).every(Boolean);
+                    npaFlag = Object.values(state.tcfv2.tcfData).includes(false);
```

Comment:
NOTA BENE: If we're talking about purposes, note that in case all purposes are false, the purpose object will be empty. This is explained in details here: https://trello.com/c/U5hiF9vd/816-cmp-problem-gettcdata-returns-an-incorrect-consent-object#comment-5ee3fd89dc12915ef5f7027c


In short, in case of no consent at all, the Object.values( consentObject ) will be zero sized, so it will NOT include `false` as a value.

Reply:
Thanks for the insights @jeteve ! I have now updated it to check if it's an empty object OR has at least one false purpose